php/datasource/class-fieldmanager-datasource.php

```diff
@@ -7,66 +12,87 @@
  *
  * This class can be used as an arbitrary data source for static options, or it
  * can be extended to provide custom data sources.
- *
- * @package Fieldmanager_Datasource
  */
 class Fieldmanager_Datasource {
 
 	/**
+	 * The datasource options.
+	 *
 	 * @var array
 	 */
 	public $options = array();
 
 	/**
+	 * Options callback.
+	 *
 	 * @var boolean
```

Comment:
I think this is supposed to be a `callable`? But as far as I can tell it's used only in `get_ajax_action()`.

Reply:
I have absolutely no idea what this is or is used for. But it seems to be concatenated, so I guess it should be a string? /shrug.